http/task_service.go

```diff
@@ -1184,7 +1190,7 @@ func (t TaskService) FindTasks(ctx context.Context, filter platform.TaskFilter)
 		val.Add("after", filter.After.String())
 	}
 	if filter.Organization != nil {
-		val.Add("organization", filter.Organization.String())
```

Comment:
Perhaps add name to TaskFilter allowing one to search for tasks with name?  (could be follow-on work)

Reply:
Yah, I think something like that would be better one in follow on.

Reply:
Here's the issue for it:
https://github.com/influxdata/influxdb/issues/12170